ui/qt-lib/src/Ariadne/UI/Qt/Widgets/Dialogs/ConfirmSend.hs

```diff
@@ -0,0 +1,96 @@
+module Ariadne.UI.Qt.Widgets.Dialogs.ConfirmSend
+  ( runConfirmSend
+  , ConfirmationResult(..)
+  ) where
+
+import Control.Lens (makeLensesWith)
+
+import qualified Data.Text as T
+
+import Graphics.UI.Qtah.Signal (connect_)
+
+import qualified Graphics.UI.Qtah.Widgets.QAbstractButton as QAbstractButton
+import qualified Graphics.UI.Qtah.Widgets.QBoxLayout as QBoxLayout
+import qualified Graphics.UI.Qtah.Widgets.QCheckBox as QCheckBox
+import qualified Graphics.UI.Qtah.Widgets.QDialog as QDialog
+import qualified Graphics.UI.Qtah.Widgets.QHBoxLayout as QHBoxLayout
+import qualified Graphics.UI.Qtah.Widgets.QLabel as QLabel
+import qualified Graphics.UI.Qtah.Widgets.QPushButton as QPushButton
+import qualified Graphics.UI.Qtah.Widgets.QWidget as QWidget
+
+import Ariadne.UI.Qt.Face
+import Ariadne.UI.Qt.UI
+import Ariadne.UI.Qt.Widgets.Dialogs.Util
+import Ariadne.UIConfig
+import Ariadne.Util
+
+data ConfirmSend =
+  ConfirmSend
+    { dialog :: QDialog.QDialog
+    , isSure :: QCheckBox.QCheckBox
+    , confirmButton :: QPushButton.QPushButton
+    }
+
+data ConfirmationResult
+  = ConfirmationAccepted
+  | ConfirmationCanceled
+
+makeLensesWith postfixLFields ''ConfirmSend
+
+initConfirmSend :: [UiConfirmSendInfo] -> IO ConfirmSend
+initConfirmSend confirmInfo = do
+  dialog <- QDialog.new
+  QWidget.resizeRaw dialog 776 266
+  QWidget.setWindowTitle dialog (toString sendHeaderMessage)
+
+  layout <- createLayout dialog
+
+  label <- QLabel.newWithText ("SEND" :: String)
+  addHeader layout label
+  let infos = T.intercalate ", " $ map (\UiConfirmSendInfo{..} ->
+        "<b>" <> csiAmount <> " " <> csiCoin <> "</b> to <b>" <> csiAddress <> "</b>") confirmInfo
```

Comment:
I don't like that `confirmInfo` remains on the same line, maybe try putting it on the next?